installing/installing_with_agent_based_installer/prepare-pxe-infra-agent.adoc

```diff
@@ -0,0 +1,38 @@
+:_content-type: ASSEMBLY
+[id="prepare-pxe-infra-agent"]
+= Preparing PXE booting infrastructure for {product-title}
```

Comment:
We are only creating the PXE assets or files and not setting up the PXE network infrastructure. Users need to make sure their environment supports the PXE boot and that they have the necessary networking/infrastructure setup for PXE.